cypress_test/integration_tests/desktop/impress/apply_paragraph_props_text_spec.js

```diff
@@ -91,7 +91,8 @@ describe(['tagdesktop', 'tagnextcloud', 'tagproxy'], 'Apply paragraph properties
 			.should('have.attr', 'ooo:numbering-type', 'number-style');
 	});
 
-	it('Increase/decrease spacing of selected text.', function() {
+	// FIXME: fails on 6600 might be related to recent core margin updates
```

Comment:
I assume you mean one of the 4 patches at https://gerrit.libreoffice.org/c/core/+/183820/3 -- if so, let's get this in, but I would like to get Justin to try to reproduce the problem & re-enable the test in some way.

Reply:
yes, I already previously added in the same file tolerance of 5 px diff, because I got inconsistent results in some coordinates

Reply:
@jstnlth , that is :-)